schema/bitbucket_server.schema.json

```
@@ -59,6 +59,17 @@
       "pattern": "^-----BEGIN CERTIFICATE-----\n",
       "examples": ["-----BEGIN CERTIFICATE-----\n..."]
     },
+    "webhooks": {
+      "description": "Configuration for Bitbucket Server Sourcegraph plugin webhooks",
+      "type": "object",
+      "properties": {
+        "secret": {
+          "description": "Secret for authenticating incoming webhook paylods",
```

Can we add more detail to this description (i.e. similar to what we have on the Github external service)

```suggestion
          "description": "Secret for authenticating incoming webhook payloads",
```

---

I will address this in a follow up PR.